app/components/forms/fake-checkbox.tsx

```diff
@@ -25,7 +31,14 @@ export const FakeCheckbox = ({ checked }: { checked: boolean }) =>
       fill="none"
       xmlns="http://www.w3.org/2000/svg"
     >
-      <rect x="0.5" y="0.5" width="19" height="19" rx="5.5" fill="white" />
+      <rect
+        x="0.5"
+        y="0.5"
+        width="19"
+        height="19"
+        rx="5.5"
+        fill={disabled ? "#F2F4F7" : "white"}
```

Comment:
What we usually do in this situation is use the value `currentColor` for the fill prop so we dont have to manage props specifically in the svg, but we can just do it in the parent component and that controls the color of the svg.